content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -252,13 +256,14 @@ For more information on how to configure dashboard providers, refer to [Dashboar
    "timezone": "browser",
    "title": "Cluster",
    "version": 0
-}
-```
+   }
+   ```
 
-- Restart Grafana to provision the new dashboard (or wait 10 seconds for Grafana to automatically create the dashboard).
-- In the sidebar, hover the cursor over **Dashboards** (squares) icon, and click **Manage**. The dashboard appears in a **Services** folder.
+1. Restart Grafana to provision the new dashboard or wait 10 seconds for Grafana to automatically create the dashboard.
+1. In the sidebar, hover the cursor over **Dashboards** (squares) icon, and thenclick **Manage**. The dashboard appears in a **Services** folder.
+((I replaced the options.path with C:\Program Files\GrafanaLabs\grafana\public\dashboards in my JSON file, and this did not work for me. I might have the file in the wrong place, but it is the only "dashboards" folder in the grafana directory, so more likely that the path is the wrong format.))
 
-> If you don't specify an `id` in the dashboard definition, Grafana assigns one during provisioning. You can set the `id` yourself if you want to reference the dashboard from other dashboards. Be careful to not use the same `id` for multiple dashboards, as this will cause a conflict.
+> If you don't specify an `id` in the dashboard definition, then Grafana assigns one during provisioning. You can set the `id` yourself if you want to reference the dashboard from other dashboards. Be careful to not use the same `id` for multiple dashboards, as this will cause a conflict. ((I have no idea where I would do this in the file. I see two different ID fields, and one has a "2" in it already.))
```

Comment:
I'm referring to the top-level `id` here, which refers to the dashboard. The `"id": 2` is the id of the panel.

Reply:
I think this might be a little far down in the weeds for a tutorial unless you are going to have them add another dashboard as part of the tutorial and then change the id.